wicket/src/wicketd.rs

```diff
@@ -153,6 +163,41 @@ impl WicketdManager {
         });
     }
 
+    fn clear_update_state(
+        &self,
+        component_id: ComponentId,
+        options: ClearUpdateStateOptions,
+    ) {
+        let log = self.log.clone();
+        let addr = self.wicketd_addr;
+        let events_tx = self.events_tx.clone();
+        tokio::spawn(async move {
+            let update_client =
+                create_wicketd_client(&log, addr, WICKETD_TIMEOUT);
```

Comment:
This wasn't changed as part of this PR, but I bumped into it while testing it. I suspended wicketd and hit the "clear update state" to see what would happen with an unresponsive wicketd, and I got a timeout error after just 1 second, which tracks back to `WICKETD_TIMEOUT` here. Some random thoughts:

* 1 second is probably too aggressive even for wicket -> wicket being on localhost. Maybe 5-10 seconds?
* I'm a little surprised we haven't seen problems due to this.
* Is it possible we actually have, particularly #3103? Could we trigger that "400 no body" response from MGS's hyper if we timed out while we were still sending a request somehow?

Reply:
That's a good hypothesis. I've updated the wicket timeout to 5 seconds and added a comment about that.